plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -691,7 +691,7 @@ public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle ha
                 Optional.ofNullable(handle.getCatalogName()),
```

Comment:
What is the motivation behind this refactoring? 

Can we make this PR small and if refactoring is independent can we extract it to the separate PR?